core/vm/vm.go

```diff
@@ -44,14 +54,24 @@ func (self *Vm) Run(context *Context, input []byte) (ret []byte, err error) {
 		value  = context.value
 		price  = context.Price
 
-		op       OpCode                  // current opcode
-		codehash = crypto.Sha3Hash(code) // codehash is used when doing jump dest caching
-		mem      = NewMemory()           // bound memory
-		stack    = newstack()            // local stack
-		statedb  = self.env.State()      // current state
+		op       OpCode                 // current opcode
+		codehash = crypto.FnvHash(code) // codehash is used when doing jump dest caching
```

Comment:
Probably not too important now, but for this case and similar ones, we could use Murmur3 to get even better speed: https://code.google.com/p/smhasher/wiki/MurmurHash3#Bulk_speed_test,_hashing_an_8-byte-aligned_256k_block